src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Apprentice/DetailsPartials/_ActionRequiredBanner.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿@inject ILinkGenerator LinkGenerator
+@using SFA.DAS.CommitmentsV2.Types
+@using SFA.DAS.EmployerCommitmentsV2.Web.Extensions
+@using SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice
+@model SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice.ApprenticeshipDetailsRequestViewModel
+
+@{ 
+   var actionRequiredBanners = Model.GetActionRequiredBanners();
+}
+
+@if (actionRequiredBanners > ActionRequiredBanner.None)
+{
+    <div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+        <div class="govuk-notification-banner__header">
+            <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+                Action required
+            </h2>
+        </div>
+        <div class="govuk-notification-banner__content">
+            @if (actionRequiredBanners.HasFlag(ActionRequiredBanner.PendingChangeForApproval))
```

Comment:
HasFlagCheck could go inside `<p>` tag

Reply:
That would produce an empty <p> tag when the check was not met